src/main/java/org/humancellatlas/ingest/core/MetadataDocumentMessageBuilder.java

```diff
@@ -124,4 +135,10 @@ public ExportMessage buildAssaySubmittedMessage() {
                 documentType.getSimpleName(), envelopeId, envelopeUuid, assayIndex, totalAssays);
     }
 
+    public ExportMessage buildUpdateExportMessage(BundleManifest bundleManifest) {
```

Comment:
This needs to take a Collection<MetadataDocument>, or Collection<String> as an additional param. Exporter needs to know which documents in the given Bundle to update